packages/hash/frontend/src/components/BlockContextMenu/BlockLoaderInput.tsx

```diff
@@ -24,6 +34,10 @@ export const BlockLoaderInput: React.VFC = () => {
       .then((blockMeta) => {
         setError(null);
         setBlocksMeta((prev) => ({ ...prev, [blockUrl]: blockMeta }));
+        setUserBlocks((prevUserBlocks) => [
+          ...prevUserBlocks,
+          blockMeta.componentMetadata as UserBlock,
+        ]);
```

Comment:
I *think* `blockMeta` and `userBlocks` are possibly duplicative? There might be a minor difference in them in certain circumstances, but I don't think that justifies entirely separate datasets. I think we need to find a way to combine them, because reasoning about the differences now is quite confusing.

Reply:
Talked offline – agreed to remove `BlocksMeta` provider